api/groups/nodeGroup.go

```diff
@@ -338,6 +359,67 @@ func (ng *nodeGroup) connectedPeersRatings(c *gin.Context) {
 	)
 }
 
+// managedKeysCount returns the node's number of managed keys
+func (ng *nodeGroup) managedKeysCount(c *gin.Context) {
+	count := ng.getFacade().GetManagedKeysCount()
+	c.JSON(
+		http.StatusOK,
+		shared.GenericAPIResponse{
+			Data:  gin.H{"count": count},
+			Error: "",
+			Code:  shared.ReturnCodeSuccess,
+		},
+	)
+}
+
+// managedKeysEligible returns the node's eligible managed keys
+func (ng *nodeGroup) managedKeysEligible(c *gin.Context) {
+	epoch, err := getQueryParamEpoch(c)
+	if err != nil {
+		shared.RespondWithValidationError(c, errors.ErrValidation, errors.ErrBadUrlParams)
+		return
+	}
+
+	keys, err := ng.getFacade().GetEligibleManagedKeys(epoch)
```

Comment:
this works for older epochs but won't work for future epochs. Do you think it is relevant to keep the epoch parameter? I think the use case will be to check the current epoch status.

Reply:
right, would make more sense.. updated